requirements.txt

```diff
@@ -18,3 +18,5 @@ python-socketio==5.0.4
 -e .
 toml==0.10.2
 Werkzeug==1.0.1
+
+python3-opencv==4.2.0
```

Comment:
just use `pip freeze > requirements.txt`

don't forget to reset line 18 to `-e .`, because the pip freeze will replace it with absolute path